src/main/java/com/fasterxml/jackson/annotation/JacksonInject.java

```diff
@@ -93,25 +109,28 @@ public static Value empty() {
             return EMPTY;
         }
 
-        public static Value construct(Object id, Boolean useInput) {
```

Comment:
For backwards compatibility, must not remove old factory method (since something may be calling it); just `@deprecate` old, make call new one with `null` for "optional".
And add Javadoc with `@since 2.20` for new factory method.

Changing constructor ok since it's not `public`.